### DISCUSSION TITLE
Does AP Manager work on a device that is already registered in Autopilot?

### DISCUSSION THREAD

**Discussion by ltorreslt** [okieselbach/Autopilot-Manager#7] [Q&A] [answered]
Oliver, awesome application. My question is if a device is already registered will the AP Manger app still group tag it and add it the all the appropriate group(s)? In my testing it appears to work great on unregistered devices, but when I test it against a registered device it doesn't always add the group tag and add it to the deployment profile group. It may very well be that I'm misunderstanding the functionality of the application in the first place.

**Comment by okieselbach** [comment, marked as answer]
Hey ltorreslt,
it is actually designed to work for unregistered devices. Devices already Autopilot registered where never my targeted use case. So, I did not implement a logic to do just tagging for already registered ones.
best,
Oliver

**Comment by ltorreslt** [comment]
Understood. Thank you for your quick response.

Lou

Luis Torres
Public Computing Services
Yale Client Technology Services
Yale University
Email: ***@***.******@***.***>
Office: (203) 737-1652

NOTE: Yale ITS will NEVER request
passwords or other personal information
via email. If you receive fraudulent
Messages requesting such information,
please DO NOT click on any links contained
inside the fraudulent message and delete
the message.



________________________________
